src/app/pages/sharing/smb/smb-form/smb-form.component.ts

```diff
@@ -44,7 +44,13 @@ export class SMBFormComponent implements OnDestroy {
       placeholder: T('Use as home share'),
       tooltip: T('Set to allow this share to hold user home\
                   directories. Only one share can be\
-                  the <i>homes</i> share.')
+                  the <i>homes</i> share.\
+                  <b>Note:</b> Samba converts %u to the lowercase\
```

Comment:
This is confusing to the end user.  Where does %u come from?  It has not been mentioned elsewhere.